packages/dtos/src/job.ts

```diff
@@ -235,6 +179,10 @@ export class CreateJobDto {
   shooting: ShootingDto[]
 }
 
+export class EditJobDto extends CreateJobDto implements EditJobType {
```

Comment:
CreateJobDto should be implemented the type instead, If you put it here there are no need to use it. the implements type is use for type checking with the database only